app/views/conversations/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @conversations.each do |conversation| %>
+    <li>
+      <%= link_to "#{conversation.job.title} - #{conversation.with.display_name}", messages_path(job: conversation.job.id, with: conversation.with.id) %>
```

Comment:
Looks like this will give the requester something like `fix my toilet - generic plumbing company` while a professional sees `fix my toilet - john smith`